<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PhoneInfoWidget</class>
 <widget class="QWidget" name="PhoneInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>842</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>749</width>
     <height>507</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="layoutPhoneInfo">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <property name="horizontalSpacing">
     <number>8</number>
    </property>
    <property name="verticalSpacing">
     <number>4</number>
    </property>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="labelModel">
      <property name="text">
       <string>Model</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2" colspan="3">
     <widget class="QLineEdit" name="lineEditModel">
      <property name="text">
       <string notr="true"/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLabel" name="labelOperator">
      <property name="text">
       <string>Operator</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2" colspan="3">
     <widget class="QLineEdit" name="lineEditOperator">
      <property name="text">
       <string notr="true"/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QLabel" name="labelSerialNumber">
      <property name="text">
       <string>Serial number</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2" colspan="3">
     <widget class="QLineEdit" name="lineEditSerialNumber">
      <property name="text">
       <string notr="true"/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QLabel" name="labelBootloader">
      <property name="text">
       <string>Bootloader</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2" colspan="3">
     <widget class="QLineEdit" name="lineEditBootloader">
      <property name="text">
       <string notr="true"/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QLabel" name="labelRom">
      <property name="text">
       <string>Rom</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2" colspan="3">
     <widget class="QLineEdit" name="lineEditRom">
      <property name="text">
       <string notr="true"/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="7" column="0" rowspan="2" colspan="2">
     <widget class="QLabel" name="labelRadio">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="text">
       <string>Radio</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2" rowspan="2" colspan="3">
     <widget class="QLineEdit" name="lineEditRadio">
      <property name="text">
       <string notr="true"/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="8" column="5" colspan="2">
     <widget class="QLabel" name="labelBatteryLevel">
      <property name="text">
       <string>Battery level</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
     </widget>
    </item>
    <item row="10" column="0" rowspan="7">
     <widget class="QProgressBar" name="progressBarData">
      <property name="minimumSize">
       <size>
        <width>41</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QProgressBar {
border: 1px solid black;
text-align: top;
padding: 1px;
border-bottom-right-radius: 7px;
border-bottom-left-radius: 7px;
background: QLinearGradient( x1: 0, y1: 0, x2: 1, y2: 0,
stop: 0 #fff,
stop: 0.4999 #eee,
stop: 0.5 #ddd,
stop: 1 #eee );
width: 18px;
}
QProgressBar::chunk {
background: QLinearGradient( x1: 0, y1: 0, x2: 1, y2: 0,
stop: 0 #78d,
stop: 0.4999 #46a,
stop: 0.5 #45a,
stop: 1 #238 );
border-bottom-right-radius: 7px;
border-bottom-left-radius: 7px;
border: 1px solid black;
}</string>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="format">
       <string notr="true">%p%</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1" colspan="2">
     <widget class="QLabel" name="label">
      <property name="text">
       <string notr="true">/data</string>
      </property>
     </widget>
    </item>
    <item row="10" column="3" rowspan="7">
     <widget class="QProgressBar" name="progressBarSystem">
      <property name="minimumSize">
       <size>
        <width>41</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QProgressBar {
border: 1px solid black;
text-align: top;
padding: 1px;
border-bottom-right-radius: 7px;
border-bottom-left-radius: 7px;
background: QLinearGradient( x1: 0, y1: 0, x2: 1, y2: 0,
stop: 0 #fff,
stop: 0.4999 #eee,
stop: 0.5 #ddd,
stop: 1 #eee );
width: 18px;
}
QProgressBar::chunk {
background: QLinearGradient( x1: 0, y1: 0, x2: 1, y2: 0,
stop: 0 #78d,
stop: 0.4999 #46a,
stop: 0.5 #45a,
stop: 1 #238 );
border-bottom-right-radius: 7px;
border-bottom-left-radius: 7px;
border: 1px solid black;
}</string>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="format">
       <string notr="true">%p%</string>
      </property>
     </widget>
    </item>
    <item row="10" column="4" colspan="2">
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string notr="true">/system</string>
      </property>
     </widget>
    </item>
    <item row="10" column="6" rowspan="7">
     <widget class="QProgressBar" name="progressBarSdcard">
      <property name="minimumSize">
       <size>
        <width>41</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QProgressBar {
border: 1px solid black;
text-align: top;
padding: 1px;
border-bottom-right-radius: 7px;
border-bottom-left-radius: 7px;
background: QLinearGradient( x1: 0, y1: 0, x2: 1, y2: 0,
stop: 0 #fff,
stop: 0.4999 #eee,
stop: 0.5 #ddd,
stop: 1 #eee );
width: 18px;
}
QProgressBar::chunk {
background: QLinearGradient( x1: 0, y1: 0, x2: 1, y2: 0,
stop: 0 #78d,
stop: 0.4999 #46a,
stop: 0.5 #45a,
stop: 1 #238 );
border-bottom-right-radius: 7px;
border-bottom-left-radius: 7px;
border: 1px solid black;
}</string>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="format">
       <string notr="true">%p%</string>
      </property>
     </widget>
    </item>
    <item row="10" column="7">
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string notr="true">/sdcard</string>
      </property>
     </widget>
    </item>
    <item row="10" column="8" rowspan="7">
     <widget class="QProgressBar" name="progressBarExt">
      <property name="minimumSize">
       <size>
        <width>41</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QProgressBar {
border: 1px solid black;
text-align: top;
padding: 1px;
border-bottom-right-radius: 7px;
border-bottom-left-radius: 7px;
background: QLinearGradient( x1: 0, y1: 0, x2: 1, y2: 0,
stop: 0 #fff,
stop: 0.4999 #eee,
stop: 0.5 #ddd,
stop: 1 #eee );
width: 18px;
}
QProgressBar::chunk {
background: QLinearGradient( x1: 0, y1: 0, x2: 1, y2: 0,
stop: 0 #78d,
stop: 0.4999 #46a,
stop: 0.5 #45a,
stop: 1 #238 );
border-bottom-right-radius: 7px;
border-bottom-left-radius: 7px;
border: 1px solid black;
}</string>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="format">
       <string notr="true">%p%</string>
      </property>
     </widget>
    </item>
    <item row="10" column="9">
     <widget class="QLabel" name="labelExt">
      <property name="text">
       <string notr="true">/sdext</string>
      </property>
     </widget>
    </item>
    <item row="11" column="1" colspan="2">
     <widget class="QLabel" name="labelDataAvailable">
      <property name="text">
       <string>available:</string>
      </property>
     </widget>
    </item>
    <item row="11" column="4" colspan="2">
     <widget class="QLabel" name="labelSystemAvailable">
      <property name="text">
       <string>available:</string>
      </property>
     </widget>
    </item>
    <item row="11" column="7">
     <widget class="QLabel" name="labelSdcardAvailable">
      <property name="text">
       <string>available:</string>
      </property>
     </widget>
    </item>
    <item row="11" column="9">
     <widget class="QLabel" name="labelExtAvailable">
      <property name="text">
       <string>available:</string>
      </property>
     </widget>
    </item>
    <item row="12" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEditDataAvailable">
      <property name="text">
       <string notr="true"/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="12" column="4" colspan="2">
     <widget class="QLineEdit" name="lineEditSystemAvailable">
      <property name="text">
       <string notr="true"/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="12" column="7">
     <widget class="QLineEdit" name="lineEditSdcardAvailable">
      <property name="text">
       <string notr="true"/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="12" column="9">
     <widget class="QLineEdit" name="lineEditExtAvailable">
      <property name="text">
       <string notr="true"/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="13" column="1" colspan="2">
     <widget class="QLabel" name="labelDataUsed">
      <property name="text">
       <string>used:</string>
      </property>
     </widget>
    </item>
    <item row="13" column="4" colspan="2">
     <widget class="QLabel" name="labelSystemUsed">
      <property name="text">
       <string>used:</string>
      </property>
     </widget>
    </item>
    <item row="13" column="7">
     <widget class="QLabel" name="labelSdcardUsed">
      <property name="text">
       <string>used:</string>
      </property>
     </widget>
    </item>
    <item row="13" column="9">
     <widget class="QLabel" name="labelExtUsed">
      <property name="text">
       <string>used:</string>
      </property>
     </widget>
    </item>
    <item row="14" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEditDataUsed">
      <property name="text">
       <string notr="true"/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="14" column="4" colspan="2">
     <widget class="QLineEdit" name="lineEditSystemUsed">
      <property name="text">
       <string notr="true"/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="14" column="7">
     <widget class="QLineEdit" name="lineEditSdcardUsed">
      <property name="text">
       <string notr="true"/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="14" column="9">
     <widget class="QLineEdit" name="lineEditExtUsed">
      <property name="text">
       <string notr="true"/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="15" column="1" colspan="2">
     <widget class="QLabel" name="labelDataSize">
      <property name="text">
       <string>size:</string>
      </property>
     </widget>
    </item>
    <item row="15" column="4" colspan="2">
     <widget class="QLabel" name="labelSystemSize">
      <property name="text">
       <string>size:</string>
      </property>
     </widget>
    </item>
    <item row="15" column="7">
     <widget class="QLabel" name="labelSdcardSize">
      <property name="text">
       <string>size:</string>
      </property>
     </widget>
    </item>
    <item row="15" column="9">
     <widget class="QLabel" name="labelExtSize">
      <property name="text">
       <string>size:</string>
      </property>
     </widget>
    </item>
    <item row="16" column="4" colspan="2">
     <widget class="QLineEdit" name="lineEditSystemSize">
      <property name="text">
       <string notr="true"/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="16" column="7">
     <widget class="QLineEdit" name="lineEditSdcardSize">
      <property name="text">
       <string notr="true"/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="16" column="9">
     <widget class="QLineEdit" name="lineEditExtSize">
      <property name="text">
       <string notr="true"/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="5" rowspan="6" colspan="2">
     <widget class="QProgressBar" name="progressBarBatteryLevel">
      <property name="minimumSize">
       <size>
        <width>87</width>
        <height>0</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QProgressBar {
border: 1px solid black;
text-align: center;
padding: 1px;
border-bottom-right-radius: 7px;
border-bottom-left-radius: 7px;
background: QLinearGradient( x1: 0, y1: 0, x2: 1, y2: 0,
stop: 0 #fff,
stop: 0.4999 #eee,
stop: 0.5 #ddd,
stop: 1 #eee );
width: 18px;
}
QProgressBar::chunk {
background: QLinearGradient( x1: 0, y1: 0, x2: 1, y2: 0,
stop: 0 #78d,
stop: 0.4999 #46a,
stop: 0.5 #45a,
stop: 1 #238 );
border-bottom-right-radius: 7px;
border-bottom-left-radius: 7px;
border: 1px solid black;
}</string>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
      <property name="format">
       <string notr="true">%p%</string>
      </property>
     </widget>
    </item>
    <item row="16" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEditDataSize">
      <property name="text">
       <string notr="true"/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="19" column="1">
     <spacer name="verticalSpacer_90">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="14" column="10">
     <spacer name="horizontalSpacer_65">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="9" column="0" colspan="10">
     <spacer name="verticalSpacer_10">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>15</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="5">
     <spacer name="verticalSpacer_50">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>5</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="17" column="9">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>refresh</string>
      </property>
      <property name="icon">
       <iconset resource="../zasoby.qrc">
        <normaloff>:/icons/refresh.png</normaloff>:/icons/refresh.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../zasoby.qrc"/>
 </resources>
 <connections/>
</ui>
